### PR TITLE
Update scripting guide

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: docs/src/main/asciidoc/scripting.adoc [hunk 1]
```diff
@@ -33,7 +33,6 @@ Normally we would link to a Git repository to clone but in this case there is no
 //DEPS io.quarkus.platform:quarkus-bom:{quarkus-version}@pom
 //DEPS io.quarkus:quarkus-resteasy-reactive
 //JAVAC_OPTIONS -parameters
-//JAVA_OPTIONS -Djava.util.logging.manager=org.jboss.logmanager.LogManager
```
Comment (gsmet) [inline docs/src/main/asciidoc/scripting.adoc:36]: Why did you remove that? I think we need it?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/scripting.adoc (hunk 1 above)
Comment (zakkak) [reply docs/src/main/asciidoc/scripting.adoc:36]: `Logger` is not used in this code segment.

When used the guide specifically says that this line should be added see https://github.com/quarkusio/quarkus/pull/32028/files/8032ac025e82c5432520e330b22e124c851c37c3#diff-850f60be00e79955e6ac10aca753c6bc8f0d5c4f889860706983899c96ef2466R375.

Comment (gsmet) [reply docs/src/main/asciidoc/scripting.adoc:36]: Even if you don't use a logger in your script, you have the standard Quarkus loggers logging things and they need this.

Comment (zakkak) [reply docs/src/main/asciidoc/scripting.adoc:36]: > you have the standard Quarkus loggers logging things and they need this

According to the Quarkus docs:

https://github.com/quarkusio/quarkus/blob/21d598d70a9e62b619d69b3ddb95622af3755671/docs/src/main/asciidoc/logging.adoc?plain=1#L13

Comment (maxandersen) [reply docs/src/main/asciidoc/scripting.adoc:36]: the property is still recommended to set as otherwise your logging configuration is not going to take effect on the first lines.

File: docs/src/main/asciidoc/scripting.adoc [hunk 2]
```diff
@@ -149,9 +147,11 @@ For now the application does nothing new.
 .How do I edit this file and get content assist?
 ====
 As there is nothing but a `.java` file, most IDE's don't handle content assist well.
-To work around that you can run `jbang edit quarkusapp.java`, this will print out a directory that will have a temporary project setup you can use in your IDE.
+To work around that you can run `jbang edit quarkusapp.java`.
+This will create a directory, containing a temporary project setup, and will open it in https://www.jbang.dev/documentation/guide/latest/editing.html[the default editor].
```
Comment (maxandersen) [inline docs/src/main/asciidoc/scripting.adoc:151]: in latest jbang we actually default to assume you have jbang plugin installed.

**Review by maxandersen — DISMISSED** [review]
sorry - completely missed this. fixes are good but a few adjustment nee

File: docs/src/main/asciidoc/scripting.adoc [hunk 3]
```diff
@@ -432,9 +439,9 @@ __  ____  __  _____   ___  __ ____  ______
  --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
  -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
 --\___\_\____/_/ |_/_/|_/_/|_|\____/___/
-2020-08-30 06:23:36,846 INFO  [io.quarkus] (main) Quarkus {quarkus-version} native started in 0.015s. Listening on: http://0.0.0.0:8080
-2020-08-30 06:23:36,846 INFO  [io.quarkus] (main) Profile prod activated.
-2020-08-30 06:23:36,846 INFO  [io.quarkus] (main) Installed features: [cdi, resteasy-reactive]
+2023-03-22 09:38:45,450 INFO  [io.quarkus] (main) quarkus 999-SNAPSHOT native (powered by {quarkus-version}) started in 0.009s. Listening on: http://0.0.0.0:8080
```
[the remark this answers is not in the document]
Comment (zakkak) [reply docs/src/main/asciidoc/scripting.adoc:442]: This is the application version, not the quarkus version.

i.e., running the example above with 3.2.0.Final results in the following output:

```
2023-07-10 14:49:08,347 INFO  [io.quarkus] (main) quarkus 999-SNAPSHOT on JVM (powered by Quarkus 3.2.0.Final) started in 0.289s. Listening on: http://0.0.0.0:8080
```

File: docs/src/main/asciidoc/scripting.adoc (hunk 1 above)
Comment (zakkak) [reply docs/src/main/asciidoc/scripting.adoc]: I added it back.

**Comment by zakkak** [comment]
> sorry - completely missed this. fixes are good but a few adjustment nee

@maxandersen could you please have another look?

**Review by gsmet — APPROVED** [review]
All the comments were addressed.
